enterprise/functions/src/test/java/io/crate/window/LastValueFunctionTest.java

```diff
@@ -73,4 +75,24 @@ public void testLastValueFunctionWithOrderByClause() throws Exception {
             new Object[] {2, 3}
         );
     }
+
+    @Test
+    public void testLastValueFunctionUseSymbolMultipleTimes() throws Exception {
+        Map<ColumnIdent, Integer> mapping = new HashMap<>();
+        mapping.put(new ColumnIdent("x"), 0);
+        mapping.put(new ColumnIdent("y"), 1);
+
+        assertEvaluate("last_value(x) over(order by y, x)",
+            // cannot match an exact output
+            iterableWithSize(4),
```

Comment:
It should be possible to assert the values in any order?

Reply:
I think I've already changed that